docs/rules/require-prop-comment.md

```diff
@@ -0,0 +1,68 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/require-prop-comment
+description: require prop should have a comment
+---
+# vue/require-prop-comment
+
+> require prop should have a comment
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+
+## :book: Rule Details
+
+This rule requires that the prop should have a comment
+
+<eslint-code-block :rules="{'vue/require-prop-comment': ['error']}">
+
+```vue
+<!-- ✓ GOOD -->
+<template>
+  <div>1</div>
+</template>
+<script>
+import { defineComponent } from '@vue/composition-api'
+
+export default defineComponent({
+  props: {
+    /**
+     * a comment
+     */
+    a: Number,
+    // b comment
+    b: Number,
+    // c
+    // comment
+    c: Number
```

Comment:
Should single-line comments really be considered "good"? When documenting props is the goal, I think JSDoc comments should be preferred over single-line comments.

Probably also documentation generator tools will ignore single-line comments. What's your opinion about this?

Reply:
I think JSDoc comments should be preferred over single-line comments too.
The second parameter will provide a choice，line 、block or unlimited
I am thinking about the following questions
1.Can I mix them?
2.Need to limit the quantity?

Reply:
I think the rule should only consider JSDoc comments (`/** */`) and ignore block comments (`/* */`) and line comments `//`. No need to limit the number then, since only the JSDoc comment immediately before the statement will be considered.